tor/core/inbox.py

```diff
@@ -57,15 +57,14 @@ def check_inbox(config):
         if 'reload' in item.subject.lower():
             item.mark_read()
             reload_config(item, config)
-            item.reply(
-                'Config reloaded!'
-            )
+
             continue
         if 'update' in item.subject.lower():
             item.mark_read()
             update_and_restart(item, config)
             # there's no reason to do anything else here because the process
             # will terminate and respawn
+            # ok so that doesn't actually happen so ignore that
```

Comment:
Not sure the point of this comment. Honestly, since the update and restart item isn't working anyway, we should probably just remove this section entirely, but that's an issue for another time.

I might suggest here that we change the wording of the original comment rather than tack on more comments at the end refuting prior ones.

Reply:
I'm with @TheLonelyGhost on this one -- since it's not part of the PR, let's remove the comment and leave for later.